translate/compiler_internal_test.go

```diff
@@ -16,7 +16,24 @@
 
 package translate
 
-import "testing"
+import (
+	"bytes"
+	"reflect"
+	"testing"
+
+	"github.com/DE-labtory/koa/abi"
+	"github.com/DE-labtory/koa/ast"
+)
+
+// TODO: implement test cases :-)
+func Test_generateFuncJumper(t *testing.T) {
```

Comment:
Hmm we need to talk about test function's name.
In parser, we named like TestGenerateFuncJumper()

Reply:
Oh, i'll modify it!